profile_bluesky/startup/10-devices.py

```diff
@@ -89,4 +96,109 @@ class SlitIpinkDevice(Device):
+    z = Component(EpicsMotor, '8ida:TA1:y', labels=["motor", "table"])
+  
+
+class TA2(Device):  
+    """
+    Optics Table 2 in 8-ID-A
+    """    
+    x = Component(EpicsMotor, '8ida:TA2:x', labels=["motor", "table"])
+    z = Component(EpicsMotor, '8ida:TA2:y', labels=["motor", "table"])
+    fine = Component(EpicsMotor, '8ida:sm9', labels=["motor", "table"])
+
+
+class DiamondMirror(Device):  
+    """
+    I/E Beam-splitting Mirror in 8-ID-D
+    """    
+    x = Component(EpicsMotor, '8idd:m1', labels=["motor", "optics"])
+    z = Component(EpicsMotor, '8idd:m2', labels=["motor", "optics"])
+
+
+class TI1(Device):  
+    """
+    Optics Table 1 in 8-ID-I
+    """    
+    x = Component(EpicsMotor, '8idi:TI1:x', labels=["motor", "table"])
+    z = Component(EpicsMotor, '8idi:TI1:y', labels=["motor", "table"])
+    
+    
+class TI2(Device):  
+    """
+    Optics Table 2 in 8-ID-I
+    """    
+    x = Component(EpicsMotor, '8idi:TI2:x', labels=["motor", "table"])
+    z = Component(EpicsMotor, '8idi:TI2:y', labels=["motor", "table"])
+    
+    
+class TI3(Device):  
+    """
+    Optics Table 3 in 8-ID-I
+    """    
+    x = Component(EpicsMotor, '8idi:TI3:x', labels=["motor", "table"])
+    y = Component(EpicsMotor, '8idi:m16', labels=["motor", "table"])  # ? Why is ti3 the only one that has a y stage?
+    z = Component(EpicsMotor, '8idi:TI3:y', labels=["motor", "table"])
+    
+
+class TI4(Device):  
+    """
+    Optics Table 4 in 8-ID-I
+    """    
+    x = Component(EpicsMotor, '8idi:TI4:x', labels=["motor", "table"])
+    z = Component(EpicsMotor, '8idi:TI4:y', labels=["motor", "table"])
+    
+    
+class BeWindow(Device):  
+    """
+    Beryllium Window 8-ID-I
+    """    
+    x = Component(EpicsMotor, '8idi:m17', labels=["motor", "optics"])
+    z = Component(EpicsMotor, '8idi:m11', labels=["motor", "optics"])
+
+        
+class Shutter(Device):  
+    """
+    Shutter Stage at 8-ID-I
+    """    
+    x = Component(EpicsMotor, '8idi:m1', labels=["motor", "shutter"])
+    z = Component(EpicsMotor, '8idi:m2', labels=["motor", "shutter"])
+ 
+        
+class Shutter(Device):  
+    """
+    Shutter Stage at 8-ID-I
+    """    
+    x = Component(EpicsMotor, '8idi:m1', labels=["motor", "shutter"])
+    z = Component(EpicsMotor, '8idi:m2', labels=["motor", "shutter"])
+    
+            
+class DetStageU(Device):  
+    """
+    Upstream detector stage at 4 m (the old 'ccdx' and 'ccdz')
+    """    
+    x = Component(EpicsMotor, '8idi:m90', labels=["motor", "det"])
+    z = Component(EpicsMotor, '8idi:m91', labels=["motor", "det"])
+  
+
+class DetStageD(Device):  
+    """
+    Downstream detector stage at 8 m (the old 'fccdx' and 'fccdz')
+    """    
+    x = Component(EpicsMotor, '8idi:m25', labels=["motor", "det"])
+    z = Component(EpicsMotor, '8idi:m83', labels=["motor", "det"])
+
+    
+class Piezo(Device):  
```

Comment:
Rather generic name for stage with everything specified.  Perhaps something like `SamplePiezoStage`?